### ISSUE TITLE
Incorporate heading violation check into Tugboat content-build actions.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12396]
## Description

See #12127.

The script exists, but it's not executed regularly.  It should be executed as part of Tugboat content-build processes.

Laura said:

>a report from the day before would be fine.  We are just looking for something repeatable, predictable and historical.  So that we can easily see what impact our own changes have but more importantly when we start to make changes and get our editors involved, we want to have something  so we can say "1 month ago we had XX issues but since we did this, our editors improved and we now only have XX issues"

>The report you gave me the other day, I noticed I haven't been using the HTML as much as I thought.  I think it's because there are just so many I can't even deal with it right now - and because I specifically need to find very very specific violations so I unfortunately just need to look at things.  But I do think it will be useful in the future, but I don't think we need that much more data than what you already provided

Based on this, I think I'll focus on getting this working in Tugboat and creating a known path so that we can access the HTML report easily, but I won't bother with optimizing the HTML report for frequent and wide access or interfering with the regular content-build process.

## Acceptance Criteria
- [ ] The script:
  - [x] creates an HTML report with a fixed location on the server.
  - [x] creates a JSON file with various metrics about the build:
    - [x] total files
    - [x] total files with violations
    - [x] total violations
  - [ ] sends the metrics from the JSON file to DataDog.
  - [ ] provides a plain text summary with URLs, etc.

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`
